### ISSUE TITLE
Has the github package for Atom been abandoned?

### ISSUE THREAD

**Issue by flamein** [atom/github#2704]
The amount of open issues is stagering, and even clear spam or nonsensical issues are left for weeks.

I also see problems with how the UI behaves (or disbehaves) but can't put my finger on it what exactly goes wrong

**Comment by smashwilson** [comment]
True, I haven't had bandwidth to do even issue triage recently 😞 . I am still watching the repo, even though I haven't engaged here for a while. I'd try to make time to address things like security issues or major breakage ("this package crashes every time, for everyone, on beta") -- or at least alert the Atom core team if I couldn't -- so I watch for signs of that.

For a little context, I was a member of the team here at GitHub who originally developed and released the package and supported it in an official capacity for a few years. After we'd moved on to different teams and products for my day job I'd done maintenance work here for some time after that on my own time. Then my free time evaporated for a long time last year, and to be honest I got pretty burned out dealing with the fallout from a bug that made it into stable.

I'd still be happy to merge and review any PRs though!

The first thing that would need to be addressed before doing anything significant would be fixing our CI, which has atrophied over time. It's a little hard to get motivated to start with that before getting to any of the user-visible things though, and it's also a tough thing to ask the community to do, so here we are.

**Comment by kincaidj790** [comment]
Use atpm

**Comment by flamein** [comment]
So yeah, this happened: https://github.blog/2022-06-08-sunsetting-atom/
